build.sbt

```diff
@@ -27,4 +27,14 @@ lazy val core = Project("fixql-core", file("fixql-core"))
   .settings(name := "fixql-core")
   .settings(libraryDependencies ++= Seq(
     scalaTest % Test,
+    "io.higherkindness" %% "droste-core" % "0.6.0",
+    "com.graphql-java" % "graphql-java" % "12.0",
+    "org.scala-lang.modules" %% "scala-java8-compat" % "0.9.0",
+
+    "io.circe" %%	"circe-core" % "0.9.3",
+    "io.circe" %% "circe-parser" % "0.9.3",
+    "io.circe" %% "circe-optics" % "0.9.3",
+
+    "com.typesafe.slick" %% "slick" % "3.2.3",
+    "com.typesafe.play" %% "play-json" % "2.6.9",
```

Comment:
This is pretty ugly but in the initial implementation I'm using both Play JSON and circe:
- Play JSON is used to make it convenient to interoperate with our existing models (which all have serializers to JSON)
- Circe is used during query parsing since the Akka HTTP server (which I haven't copied over yet) implementation I copied from Sangria happened to be using Circe.

In a future PR, I'll either settle on using Circe, or possibly genericize it and remove the dependency on both.